modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-example-dpdk-line-rate_{context}"]
+= Using SR-IOV and the Node Tuning Operator to achieve a DPDK line rate
+You can use the Node Tuning Operator to configure isolated CPUs, hugepages, and a topology scheduler.
+You can then use the Node Tuning Operator with Single Root I/O Virtualization (SR-IOV) to achieve a specific Data Plane Development Kit (DPDK) line rate.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have deployed a standalone Node Tuning Operator.
++
+[NOTE]
+====
+In previous versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11 and later, this functionality is part of the Node Tuning Operator.
+====
+
+.Procedure
+. Create a `PerformanceProfile` object based on the following example:
++
+[source,yaml]
+----
+    apiVersion: performance.openshift.io/v2
```

Comment:
I just noticed that this whole example is indented. Should the top level start with no indentation and everything else align from that? Please check with an SME or QE reviewer.

Reply:
Answer: indentation not appropriate, it's script. Removed indent.